### ISSUE TITLE
Give a custom name to the custom connectivity profile

### ISSUE THREAD

**Issue by jpvincent** [sitespeedio/sitespeed.io#1329]
Hi

Currently we can define our own connectivity limit and that is awesome

```json
{  "browsertime.connectivity.profile":"custom",
  "browsertime.connectivity.downstreamKbps":"4000",
  "browsertime.connectivity.upstreamKbps":"1000",
  "browsertime.connectivity.latency":"60"
}
```
It appears in the grafana UI and is used to filter the results
![image](https://cloud.githubusercontent.com/assets/427625/20387128/f50b489a-acbe-11e6-8fe8-131ef5431dee.png)

However, I would need more than only one "custom" connection setting, because the default ones (cable, 3gslow, 3gfast and so on) are not realistic for my users.

Could we add a custom name ?
Like : 
```json
{  "browsertime.connectivity.profile":"custom",
   "browsertime.connectivity.alias":"French ADSL",
   …
}
```

The alias would be sent to graphite with the results instead of the current "custom" string and would allow Grafana to filter the results

**Comment by soulgalore** [comment]
Hey @jpvincent I see your point, it sounds like a good idea.

**Comment by soulgalore** [comment]
I've pushed a branch in Browsertime and then when we do the next beta I'll add it and release a new version. Thanks for the good idea @jpvincent !

**Comment by softwareklinic** [comment]
Also, 3gfast is kind of outdated .. 4g LTE, 4g LTE Advanced, 5G (upcoming) --- we need to have these as some of the defaults

**Comment by jpvincent** [comment]
@softwareklinic I dont know about outdated, but from country to country, we dont have the same definition of "3G" or even Cable : I always had to modify the "Cable" definition of Webpagetest in order to match more realistic connectivity in my country (France).

@soulgalore you're welcome, kudos to the huge amout of work you're doing !

**Comment by softwareklinic** [comment]
Actually you are right

Sent from my iPhone

On Nov 18, 2016, at 4:28 PM, jpvincent <notifications@github.com<mailto:notifications@github.com>> wrote:


@softwareklinic<https://github.com/softwareklinic> I dont know about outdated, but from country to country, we dont have the same definition of "3G" or even Cable : I always had to modify the "Cable" definition of Webpagetest in order to match more realistic connectivity in my country (France).

@soulgalore<https://github.com/soulgalore> you're welcome, kudos to the huge amout of work you're doing !

-

**Comment by soulgalore** [comment]
thanks @jpvincent :) 

For me connectivity is a way of making it easier to catch regression, so it doesn't have to be "real" values. Slowing down the connection is good because it's makes it easier to changes. "Real" values is hard. Did you try WebPageTest when there was real 3g and 2g connections in SF and somewhere in India (was it Bangalore)? We ran one test every hour just to see how the speed changed depending on the hour of the day, how many users etc. I'll check if I can find some screenshots.

But Iäm not agains setting values, I'll push the change asap when we got the other things we are working on finished.
